README.rst

```diff
@@ -92,10 +92,9 @@ the ``WATCHMAN_CHECKS`` setting. In ``settings.py``::
         'another.module.path.to.callable',
     )
 
-Checks now have the same contract as context processors: they consume a
-``request`` and return a ``dict`` whose keys are applied to the JSON response::
```

Comment:
No, I didn't have any custom ones which depended directly on request, but at the time I changed it, the only usage was obviously in request/response cycle, and it seemed simple enough to alter the callables so they were familiar (in being like context processors - accept a `request`, return a `dict`). Until such a time as someone *needs* the request, it seems fine to remove that functionality.

If it does need revisiting at some point, I'd say that the management command could instantiate a fake request via `RequestFactory`, setting it's environ/defaults based on a project-level setting, or an argument to the management command to read a JSON file (which you could run through the Django Template Language if necessary) etc.